packages/editor/src/components/sidebar/post-summary.js

```diff
@@ -105,13 +103,6 @@ export default function PostSummary() {
 						<PostSlugPanel />
 						<PostAuthorPanel />
 						{ fills }
-						<HStack
-							style={ {
-								marginTop: '16px',
-							} }
-						>
-							<PostTrashPanel />
-						</HStack>
```

Comment:
I don't think we should remove this in the "unification" PRs. IMO, they should make close to zero UI/UX changes.

I know the action is available via new "Post Actions", but they're far from finished and lack [the proper checks](https://github.com/WordPress/gutenberg/pull/60754#issuecomment-2063575736).

Reply:
Not sure I agree 100%. If there's an issue with the action, we should fix it as it's already on trunk. That said, I'm fine restoring this for now if that helps unblock this PR.

Reply:
Restored in [c9349cd](https://github.com/WordPress/gutenberg/pull/61507/commits/c9349cd57601f3f9b974d81494ce10025457ebd8)

Reply:
I think having a separate issue/discussion for this UI change would be helpful. These can be shared widely and are easier to link when people ask where the decisions were made.

> If there's an issue with the action, we should fix it as it's already on trunk.

100%. I don't think the current version is ready for the next major WP beta.